functions/https/actions.js

```diff
@@ -0,0 +1,14 @@
+const express = require('express');
```

Comment:
Why add express here if you already have baseExpress?

Reply:
@kuuuurt can you check?

Reply:
@tjmonsi it would be better. updated the code accordingly